dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: 'npm'
+    # Look for `package.json` and `lock` files in the `root` directory
+    directory: '/'
+    # Check the npm registry for updates every day (weekdays)
+    schedule:
+    interval: 'daily'
```

Comment:
Probably worth setting [schedule.time ](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleday )so that it checks early in the morning or something otherwise it will check at a random time which might be a bit annoying when people are trying to merge stuff throughout the day.